### PR TITLE
Add contributor Mansi

### PR DESCRIPTION
**Pull request by mansipanchal11**

### CODE REVIEW

**Comment by beingnoble03** [comment]
@mansipanchal11 please resolve the merge conflicts.

**Comment by mansipanchal11** [comment]
> @mansipanchal11 please resolve the merge conflicts.

resolved conflicts

File: README.md [hunk 1]
```diff
@@ -14,6 +15,6 @@ Add you name and github profile here. Have fun with MERGE CONFLICTS (if you were
 - [Ayush Dhar Dubey](https://github.com/AyushDharDubey)
 - [Rayyan Khan](https://github.com/rkcoder101)
 - [Akshit Mandial](https://github.com/Akshit517)
+- [Akshit Mandial](https://github.com/Akshit517)
```
Comment (beingnoble03) [inline README.md:18]: @mansipanchal11 remove this line, this is duplicate

**Review by beingnoble03 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mansipanchal11) [reply README.md]: removed duplicate lines

**Pull request merged**